modules/component-store/src/component-store.ts

```diff
@@ -32,7 +32,9 @@ export interface SelectConfig {
   debounce?: boolean;
 }
 
-export const initialStateToken = new InjectionToken('ComponentStore InitState');
+export const INITIAL_STATE_TOKEN = new InjectionToken(
```

Comment:
Forgot about this PR.
I was thinking if this could be seen as a breaking change 🤔

Reply:
That token is part of the public API, so I would say yes

Reply:
I'll tag it for v11 :)

Reply:
ComponentStore's token was not supposed to be exposed 😐 It's necessary for ViewEngine backwards-compatibility.
I'm fine with leaving it for v11, at the same time I'm suspecting that our next release would be v11 🙂